services/migration_service/api/__init__.py

```diff
@@ -4,7 +4,8 @@
     '0': 'v_1_0_1',
     '1': 'v_1_0_1',
     '20200603104139': 'latest',
-    '20201002000616': 'latest'
+    '20201002000616': 'latest',
+    '20210202145952': 'latest'
```

Comment:
you can't have multiple versions pointing to latest, this may cause undefined behavior

Reply:
Good point, https://github.com/Netflix/metaflow-service/pull/30 this PR might have the same conflict as well.